display_list/display_list_builder.cc

```diff
@@ -67,16 +67,16 @@ sk_sp<DisplayList> DisplayListBuilder::Build() {
 }
 
 DisplayListBuilder::DisplayListBuilder(const SkRect& cull_rect,
-                                       bool prepare_rtree) {
+                                       bool prepare_rtree)
+    : tracker_(cull_rect.isEmpty() ? SkRect::MakeEmpty() : cull_rect,
+               SkMatrix::I()) {
```

Comment:
Maybe move the NaN protection to the tracker?

Reply:
Done.